### DISCUSSION TITLE
Welcome to formidablae's Discussions!

### DISCUSSION THREAD

**Discussion by formidablae** [formidablae/formidablae#31] [Announcements]
## 👋 Welcome!
  `Discussions` in this special repo is used as a place to connect, discuss and share ideas. It is possible to:
  * share ideas [here](https://github.com/formidablae/formidablae/discussions/34);
  * engage and collaborate in projects together [over here](https://github.com/formidablae/formidablae/discussions/36) 💪.
  * introduce yourself and your projects [here](https://github.com/formidablae/formidablae/discussions/32);
  * ask [here](https://github.com/formidablae/formidablae/discussions/37) the important questions you’re wondering about.

**Comment by the-sides** [comment]
Who are you? Your readme and following is insane.

Do you work anywhere?

**Reply by Arturo-K** [reply]
That readme is goals!!!!!

**Reply by formidablae** [reply]
Thank you @the-sides and @Arturo-K. I work in Milan, Italy.

**Reply by ErnestCodes** [reply]
Please how do I get my readme up to your standard

**Reply by ChrisToxz** [reply]
> Please how do I get my readme up to your standard 

Check his source :)

**Comment by nSedrickm** [comment]
Hello @formidablae happy to connect

**Reply by formidablae** [reply]
My pleasure @nSedrickm : )

**Comment by deep0072** [comment]
wow man your readme is insane. how can i connect to you?

**Reply by formidablae** [reply]
Hello Deepak :) Thank's for reaching out. You can connect with me on twitter (see handle in my profile) and on discord (see [this github discussion](https://github.com/formidablae/formidablae/discussions/36#discussion-3842454)).

**Comment by maxgio92** [comment]
Hi @formidablae!
Your README is super! I took the freedom to inspire my one from yours :-) 
I'm italian too. I also liked the idea to leverage Discussions to connect, that was a thing I didn't think about in that way.
It's cool.

**Reply by formidablae** [reply]
Thanks @maxgio92 :)
Saw your @clastix labs org and repos. Complimentoni :)
You develop products facilitating k8s cluster orchestration?
Yep, in github's special profile repo, discussions can be used to connect (in an improvised manner) ahah

**Reply by maxgio92** [reply]
Thank you @formidablae! Exactly, we develop projects specifically for multi-tenancy in Kubernetes.
What about you?

**Reply by formidablae** [reply]
That's cool, saw some of you projects :) Keep it up!
I am just a humble and curious software engineer...

**Comment by TakashiNishii** [comment]
Hello, @formidablae thank you for connect, your readme is amazing!!!

**Reply by formidablae** [reply]
My pleasure, Ygor. Yours is pretty nice too :)

**Comment by LucasColas** [comment]
Hello, nice to meet you !

**Reply by formidablae** [reply]
@LucasColas the pleasure is mine :)
